plenum/test/batching_3pc/test_basic_batching.py

```diff
@@ -88,5 +89,5 @@ def rejectingMethod(self, req, pp_time):
 
     assert 'Simulated rejection' in e._excinfo[1].args[0]
 
-    for i, node in enumerate(txnPoolNodeSet):
-        node.doDynamicValidation = origMethods[i]
+    # for i, node in enumerate(txnPoolNodeSet):
+    #     node.doDynamicValidation = origMethods[i]
```

Comment:
Why restore is commented? Btw, there is a `monkeypatch` fixture which can be also used for overriding different stuff, but it automatically restores originals in the teardown.